src/Validation/Validator.php

```diff
@@ -242,9 +247,7 @@ public function validateParams(array $formDetails, bool $strictValidation = true
 							$output[$paramKey] = $this->getValidationLabel('validationEmail', $formId);
 						} else {
 							if (UtilsSettingsHelper::isOptionCheckboxChecked(SettingsValidation::SETTINGS_VALIDATION_USE_EMAIL_TLD_KEY, SettingsValidation::SETTINGS_VALIDATION_USE_EMAIL_TLD_KEY)) {
-								$data = $this->getEmailTldData();
-
-								if (!$this->isEmailTlValid($inputValue, $data)) {
+								if (!$this->isEmailTlValid($inputValue, \array_values($this->manifestCache->getManifestCacheTopItem(ManifestCache::TLD_KEY, ManifestCache::TYPE_FORMS)))) {
```

Comment:
```suggestion
								if (!$this->isEmailTldValid($inputValue, \array_values($this->manifestCache->getManifestCacheTopItem(ManifestCache::TLD_KEY, ManifestCache::TYPE_FORMS)))) {
```

I think the `d` is missing in `Tld` part.

Reply:
you are correct, tnx